app.rb

```diff
@@ -11,6 +11,7 @@
 require 'sinatra/base'
 require 'time'
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
should we exclude this via the rubocop config file instead? It seems like that is how we've been excluding other stuff https://github.com/18F/identity-openidconnect-sinatra/blob/master/.rubocop.yml